docs/source/en/using-diffusers/kandinsky.md

```diff
@@ -0,0 +1,316 @@
+# Kandinsky
```

Comment:
Is this page supposed to include both 2.1 and 2.2? 
if so, I think we are missing the ControlNet pipelines